.circleci/ci_package.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -x
+set -e
+# TODO: The name is not always `.so`
+
+
+if [ -z "$DISTDIR" ]; then
+    DISTDIR=/workspace
+fi
+
+if [ -z "$MODULE_SO" ]; then
+    MODULE_SO=$BUILD_DIR/redisearch.so
+fi
+
+# Install my fork of RAMP- need to merge it first
+pip install git+https://github.com/mnunberg/RAMP@print_version --upgrade
+
+if [ -e /etc/redhat-release ]; then
+    DIST_SUFFIX=el7
+elif [ -e /etc/debian_version ]; then
+    DIST_SUFFIX=debian
```

Comment:
those suffixes are not as requested by Alon, Alon asked ubuntu to be left as is and RHEL to be 'RHA'